src/TripRepo.js

```diff
@@ -39,6 +41,23 @@ class TripRepo {
       return time.isBetween(trip.date, date, endDate);
     }), null, false);
   }
+
+  getNewTripID() {
```

Comment:
I didn't quite understand the functionality at first for this but realize it's importance now. Great job on this!